ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -30,7 +30,11 @@ trait ProcessRepository[F[_]] extends Repository[F] with EspTables {
       .filter(_.processId === processId.value)
       .sortBy(_.createDate.desc)
 
-  protected def fetchLastDeploymentActionPerProcess: Query[(Rep[Long], ProcessDeploymentInfoEntityFactory#ProcessDeploymentInfoEntity), (Long, DeployedProcessInfoEntityData), Seq] =
+
+  protected def fetchLastDeployedActionPerProcess: Query[(api.Rep[Long], ProcessDeploymentInfoEntityFactory#ProcessDeploymentInfoEntity), (Long, DeployedProcessInfoEntityData), Seq] =
```

Comment:
Can you change names of methods that return `Query` to be suffixed by `Query` e.g. `fetchLastDeployedActionPerProcessQuery`. I know that it was that way before, but now it looks like some fetching is done in this method but it is only building a query...

Reply:
It's good idea. I will change it.